src/fastoad/notebooks/02_From_Python_to_FAST-OAD/modules/OpenMDAO/mass/sub_components/compute_owe_exercise.py

```diff
@@ -1,5 +1,3 @@
-import numpy as np
-
```

Comment:
I'm not sure about that deletion since the users of the tutorial might have to use numpy in the formula or the declaration of inputs/outputs

Reply:
Fixed in 1116405cee9cb602b3e580b697879a946a7aa985